node-runner-cli/radixnode.py

```diff
@@ -199,6 +211,26 @@ def mempool_transaction(args):
     core_api_helper.mempool_transaction(args.transactionId, True)
 
 
+@corecommand()
+def vote(args):
+    core_api_helper = CoreApiHelper(False)
+    health = DefaultApiHelper(False).health()
+    if health['fork_vote_status'] == 'VOTE_REQUIRED':
+        candidate_fork_name = core_api_helper.engine_configuration()["forks"][-1]['name']
+        print(
+            f"{bcolors.WARNING}NOTICE: Because the validator is running software with a candidate fork ({candidate_fork_name}{bcolors.WARNING}), " +
+            "by performing this action, the validator will record support for this fork onto the ledger.\n" +
+            f"If you choose to downgrade the software to no longer support this fork, you should manually remove the validator's support for the candidate fork with the withdraw vote action.{bcolors.ENDC}"
+        )
+    core_api_helper.vote(True)
```

Comment:
Should this be inside the if? And should we have anything in the else to explain why we're not voting??